src/app/components/RelatedTopics/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { useContext } from 'react';
+import SectionLabel from '#psammead/psammead-section-label/src';
+import { GREY_2 } from '#app/components/ThemeProvider/palette';
+import { RequestContext } from '#app/contexts/RequestContext';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import useClickTrackerHandler from '#hooks/useClickTrackerHandler';
+import useViewTracker from '#hooks/useViewTracker';
+import TopicTags, { TopicTag } from '#app/components/TopicTags';
+import styles from './index.styles';
+
+const eventTrackingData = {
+  componentName: 'topics',
+};
+
+interface Topic {
+  topicName: string;
+  topicId: string;
+}
+
+interface RelatedTopicsProps {
+  topics?: Topic[];
+  mobileDivider?: boolean;
+  bar?: boolean;
+  className?: string;
+}
+
+const RelatedTopics = ({
+  topics = [],
+  mobileDivider = true,
+  bar = true,
+  className = '',
+}: RelatedTopicsProps) => {
+  const { service, script, translations, dir } = useContext(ServiceContext);
+  const { variant } = useContext(RequestContext);
+  const clickTrackerHandler = useClickTrackerHandler(eventTrackingData);
+  const viewTracker = useViewTracker(eventTrackingData);
+
+  const heading = translations?.relatedTopics ?? 'Related Topics';
+  const topicsPath = translations?.topicsPath ?? 'topics';
+
+  const getTopicPageUrl = (id: string) => {
+    const isPublicService = ['news', 'cymrufyw', 'naidheachdan'];
+    const hostname = `https://www.bbc.${isPublicService.includes(service) ? 'co.uk' : 'com'}`;
+
+    return variant
+      ? `${hostname}/${service}/${topicsPath}/${id}/${variant}`
+      : `${hostname}/${service}/${topicsPath}/${id}`;
```

Comment:
```suggestion
    return `${hostname}/${service}/${topicsPath}/${id}${variant ? `/${variant}` : ''}`;
```
Subjective preference to save a line, arguably less readable. Optional suggestion

Reply:
Resolved in https://github.com/bbc/simorgh/pull/13525/commits/9d08ea715177c665c32ccb016989f76d82b6658f